spec/tinymce_examples_spec.rb

```diff
@@ -0,0 +1,313 @@
+require_relative 'spec_helper'
+
+describe HTMLToConfluenceParser, "when consuming TinyMCE generated HTML" do
+
+  context "edge case with nested formats with spans and &nbsp;" do
+
+    it "should handle nested formatting" do
+      html = <<~HTML
+        test<i>test<b>test</b></i>
+      HTML
+
+      markup = <<~MARKUP
+        test_test*test*_
```

Comment:
Is this markup correct? When I put that markup into JIRA it is not formatted, I still see the * and _.